web/sites/default/config/core.entity_view_display.node.development_permit.default.yml

```diff
@@ -91,32 +91,11 @@ content:
     third_party_settings:
       ds:
         ft:
-          id: expert
+          id: default
           settings:
             lb: ''
-            prefix: '<span class="sr-only">Type is</span>'
-            lbw-el: ''
```

Comment:
In the few places where the field config still showed display suite expert—even though the display is now twig—I resaved the fields to clean this up. It removes some of the mb classes that were no longer functionally in use.